sources/__init__.py

```diff
@@ -11,10 +11,10 @@ def get_records():
     for query in TERMS:
         data.update(chictr.find(query))
         data.update(clinicaltrialsgov.find(query))
-        # data.update(eu.find(query))
+        data.update(eu.find(query))
         data.update(isrctn.find(query))
 
     # sort the data and return it
     items = data.values()
-    items = sorted(items, key=lambda d: float('-inf') if d["timestamp"] == None else d["timestamp"], reverse=True)
-    return items
\ No newline at end of file
+    items = sorted(items, key=lambda d: "" if not d["timestamp"] else d["timestamp"], reverse=True)
```

Comment:
Keep it as d(timestamp) == None. There are other things like not 0 and not [] that evaluate to True.